src/game/common/ini.cpp

```diff
@@ -261,46 +259,40 @@ int INIClass::Load(Straw &straw)
                         }
 
                         INIEntry *entryptr = new INIEntry(entry, value);
-                        if (!entryptr) {
+                        if (entryptr == nullptr) {
                             captainslog_debug("Failed to create entry '%s = %s'.", entry, value);
 
                             delete section;
                             Clear();
 
-                            return INI_LOAD_INVALID;
+                            return 0;
                         }
 
-                        // Is this Name, Value or something?
                         CRC(entryptr->Get_Name());
                         int32_t crc = CRC(entryptr->Get_Name());
-
-                        if (section->m_entryIndex.Is_Present(crc)) {
-                            // Duplicate_CRC_Error(__FUNCTION__, section->Get_Name(), entryptr->Get_Name());
-                        }
-
-                        section->m_entryIndex.Add_Index(crc, entryptr);
-                        section->m_entryList.Add_Tail(entryptr);
+                        section->Entry_Index().Add_Index(crc, entryptr);
+                        section->Entry_List().Add_Tail(entryptr);
                     }
                 }
             }
         }
 
-        if (section->m_entryList.Is_Empty()) {
+        if (section->Entry_List().Is_Empty()) {
             delete section;
         } else {
             int32_t crc = CRC(section->Get_Name());
-            m_sectionIndex.Add_Index(crc, section);
-            m_sectionList.Add_Tail(section);
+            m_SectionIndex.Add_Index(crc, section);
+            m_SectionList.Add_Tail(section);
         }
     }
 
-    return INI_LOAD_OVERWRITE;
+    return 1;
```

Comment:
Looks like return should be BOOL then?

Reply:
GameINIClass::Load returns '2' on a case of the digest. These return values were incorrectly added by me and are long overdue to be reverted and more research is to be done.